### ISSUE TITLE
2019-11-22 tc011 partial AUC interpolated concentration value missing in estimated concentration dataset

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#53]
Partial AUC AUC15_30 was specified as one partial AUC for tc011 and AUC15_30 is generated in results but concentration for TIME==15 doesn't appear in estimated concentration dataset

```
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2 AUC.5.T1 AUC.5.T2
1        0       12       12       24        0       48       15       30       12       26
> r$AUC15_30
 [1]  75.85705  80.95701  62.91297  73.48418  85.58079  53.56854  96.31835  80.80728  87.37859  70.64978  87.17738  85.23608 106.43399
[14] 104.75634 102.27965 105.99451  84.30598  85.79118
> e[e$TIME==15,]
[1] PKDATAROWID SDEID       TIME        CEST_KEL    CEST_INT    CEST_EXT    CEST_C0     CEST_TLAST 
<0 rows> (or 0-length row.names)
> e
    PKDATAROWID     SDEID TIME CEST_KEL CEST_INT   CEST_EXT CEST_C0 CEST_TLAST
1      26908935  70745987 18.0 4.780859       NA         NA      NA         NA
2      26908936  70745987 21.0 3.498742       NA         NA      NA         NA
3      26908937  70745987 24.0 2.560460       NA         NA      NA   2.560460
6            NA  70745987 26.0       NA       NA 2.07932333      NA         NA
5            NA  70745987 30.0       NA       NA 1.37129241      NA         NA
4            NA  70745987 48.0       NA       NA 0.21065234      NA         NA
11     26908954 130509330 18.0 4.124984       NA         NA      NA         NA
21     26908955 130509330 21.0 3.017955       NA         NA      NA         NA
31     26908956 130509330 24.0 2.208021       NA         NA      NA   2.208021
61           NA 130509330 26.0       NA       NA 1.79279271      NA         NA
51           NA 130509330 30.0       NA       NA 1.18190885      NA         NA
41           NA 130509330 48.0       NA       NA 0.18127026      NA         NA
12     26908535 246409536 16.0 5.333360       NA         NA      NA         NA
22     26908536 246409536 18.0 4.376376       NA         NA      NA         NA
32     26908537 246409536 21.0 3.253009       NA         NA      NA         NA
42     26908538 246409536 24.0 2.417997       NA         NA      NA   2.417997
7            NA 246409536 26.0       NA       NA 1.98412710      NA         NA
62           NA 246409536 30.0       NA       NA 1.33597065      NA         NA
52           NA 246409536 48.0       NA       NA 0.22533069      NA         NA
13     26908877 249265153 16.0 4.922666       NA         NA      NA         NA
23     26908878 249265153 18.0 3.780412       NA         NA      NA         NA
33     26908879 249265153 21.0 2.544177       NA         NA      NA         NA
43     26908880 249265153 24.0 1.712204       NA         NA      NA   1.712204
71           NA 249265153 26.0       NA       NA 1.31490467      NA         NA
63           NA 249265153 30.0       NA       NA 0.77548221      NA         NA
53           NA 249265153 48.0       NA       NA 0.07204783      NA         NA
14     26908516 283369224 16.0 5.622917       NA         NA      NA         NA
24     26908517 283369224 18.0 4.355208       NA         NA      NA         NA
34     26908518 283369224 21.0 2.968792       NA         NA      NA         NA
44     26908519 283369224 24.0 2.023721       NA         NA      NA   2.023721
72           NA 283369224 26.0       NA       NA 1.56746503      NA         NA
64           NA 283369224 30.0       NA       NA 0.94035608      NA         NA
54           NA 283369224 48.0       NA       NA 0.09434470      NA         NA
15     26908895 292748221 14.0 8.142487       NA         NA      NA         NA
25     26908896 292748221 16.0 6.390056       NA         NA      NA         NA
35     26908897 292748221 18.0 5.014785       NA         NA      NA         NA
45     26908898 292748221 21.0 3.486372       NA         NA      NA         NA
55     26908899 292748221 24.0 2.423791       NA         NA      NA   2.423791
8            NA 292748221 26.0       NA       NA 1.90214124      NA         NA
73           NA 292748221 30.0       NA       NA 1.17148840      NA         NA
65           NA 292748221 48.0       NA       NA 0.13227158      NA         NA
16     26908495 328228072 13.5 6.605181       NA         NA      NA         NA
26     26908496 328228072 14.0 6.128861       NA         NA      NA         NA
36     26908497 328228072 16.0 4.543182       NA         NA      NA         NA
46     26908498 328228072 18.0 3.367756       NA         NA      NA         NA
56     26908499 328228072 21.0 2.149370       NA         NA      NA         NA
66     26908500 328228072 24.0 1.371771       NA         NA      NA   1.371771
9            NA 328228072 26.0       NA       NA 1.01686217      NA         NA
81           NA 328228072 30.0       NA       NA 0.55875684      NA         NA
74           NA 328228072 48.0       NA       NA 0.03776124      NA         NA
17     26908818 331625416 13.5 9.835481       NA         NA      NA         NA
27     26908819 331625416 14.0 9.241418       NA         NA      NA         NA
37     26908820 331625416 16.0 7.202953       NA         NA      NA         NA
47     26908821 331625416 18.0 5.614131       NA         NA      NA         NA
57     26908822 331625416 21.0 3.863140       NA         NA      NA         NA
67     26908823 331625416 24.0 2.658266       NA         NA      NA   2.658266
91           NA 331625416 26.0       NA       NA 2.07190769      NA         NA
82           NA 331625416 30.0       NA       NA 1.25867712      NA         NA
75           NA 331625416 48.0       NA       NA 0.13361710      NA         NA
18     26908839 494439910 16.0 5.601851       NA         NA      NA         NA
28     26908840 494439910 18.0 4.508992       NA         NA      NA         NA
38     26908841 494439910 21.0 3.256126       NA         NA      NA         NA
48     26908842 494439910 24.0 2.351381       NA         NA      NA   2.351381
76           NA 494439910 26.0       NA       NA 1.89265243      NA         NA
68           NA 494439910 30.0       NA       NA 1.22621488      NA         NA
58           NA 494439910 48.0       NA       NA 0.17389908      NA         NA
19     26908969 543244472 13.0 8.540985       NA         NA      NA         NA
29     26908970 543244472 13.5 8.080628       NA         NA      NA         NA
39     26908971 543244472 14.0 7.645084       NA         NA      NA         NA
49     26908972 543244472 16.0 6.125348       NA         NA      NA         NA
59     26908973 543244472 18.0 4.907714       NA         NA      NA         NA
69     26908974 543244472 21.0 3.519670       NA         NA      NA         NA
77     26908975 543244472 24.0 2.524206       NA         NA      NA   2.524206
10           NA 543244472 26.0       NA       NA 2.02242879      NA         NA
92           NA 543244472 30.0       NA       NA 1.29828561      NA         NA
83           NA 543244472 48.0       NA       NA 0.17664720      NA         NA
110    26908572 695874499 14.0 7.097609       NA         NA      NA         NA
210    26908573 695874499 16.0 5.476251       NA         NA      NA         NA
310    26908574 695874499 18.0 4.225272       NA         NA      NA         NA
410    26908575 695874499 21.0 2.863595       NA         NA      NA         NA
510    26908576 695874499 24.0 1.940745       NA         NA      NA   1.940745
84           NA 695874499 26.0       NA       NA 1.49740701      NA         NA
78           NA 695874499 30.0       NA       NA 0.89142017      NA         NA
610          NA 695874499 48.0       NA       NA 0.08638218      NA         NA
111    26908476 783385801 13.5 8.322697       NA         NA      NA         NA
211    26908477 783385801 14.0 7.852061       NA         NA      NA         NA
311    26908478 783385801 16.0 6.221025       NA         NA      NA         NA
411    26908479 783385801 18.0 4.928789       NA         NA      NA         NA
511    26908480 783385801 21.0 3.475823       NA         NA      NA         NA
611    26908481 783385801 24.0 2.451179       NA         NA      NA   2.451179
93           NA 783385801 26.0       NA       NA 1.94201829      NA         NA
85           NA 783385801 30.0       NA       NA 1.21901729      NA         NA
79           NA 783385801 48.0       NA       NA 0.14993891      NA         NA
112    26908553 797620983 14.0 9.083895       NA         NA      NA         NA
212    26908554 797620983 16.0 7.349303       NA         NA      NA         NA
312    26908555 797620983 18.0 5.945936       NA         NA      NA         NA
412    26908556 797620983 21.0 4.326944       NA         NA      NA         NA
512    26908557 797620983 24.0 3.148780       NA         NA      NA   3.148780
86           NA 797620983 26.0       NA       NA 2.54751297      NA         NA
710          NA 797620983 30.0       NA       NA 1.66749484      NA         NA
612          NA 797620983 48.0       NA       NA 0.24764578      NA         NA
113    26908916 856727921 18.0 6.093222       NA         NA      NA         NA
213    26908917 856727921 21.0 4.251489       NA         NA      NA         NA
313    26908918 856727921 24.0 2.966437       NA         NA      NA   2.966437
613          NA 856727921 26.0       NA       NA 2.33362552      NA         NA
513          NA 856727921 30.0       NA       NA 1.44418641      NA         NA
413          NA 856727921 48.0       NA       NA 0.16664322      NA         NA
114    26908858 895085608 16.0 6.810659       NA         NA      NA         NA
214    26908859 895085608 18.0 5.571924       NA         NA      NA         NA
314    26908860 895085608 21.0 4.123151       NA         NA      NA         NA
414    26908861 895085608 24.0 3.051078       NA         NA      NA   3.051078
711          NA 895085608 26.0       NA       NA 2.49614286      NA         NA
614          NA 895085608 30.0       NA       NA 1.67071185      NA         NA
514          NA 895085608 48.0       NA       NA 0.27431280      NA         NA
115    26908610 897885073 14.0 8.898941       NA         NA      NA         NA
215    26908611 897885073 16.0 7.213563       NA         NA      NA         NA
315    26908612 897885073 18.0 5.847380       NA         NA      NA         NA
415    26908613 897885073 21.0 4.267550       NA         NA      NA         NA
515    26908614 897885073 24.0 3.114554       NA         NA      NA   3.114554
87           NA 897885073 26.0       NA       NA 2.52468621      NA         NA
712          NA 897885073 30.0       NA       NA 1.65893910      NA         NA
615          NA 897885073 48.0       NA       NA 0.25068832      NA         NA
116    26908593 916516839 18.0 5.022101       NA         NA      NA         NA
216    26908594 916516839 21.0 3.451510       NA         NA      NA         NA
316    26908595 916516839 24.0 2.372099       NA         NA      NA   2.372099
 [ reached 'max' / getOption("max.print") -- omitted 11 rows ]

```

**Comment by rudraya** [comment]
This issue was addressed in 18821dc, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue no longer occurs as of 18821dc. Output for est_data shown below:

```
    PKDATAROWID     SDEID TIME CEST_KEL CEST_INT   CEST_EXT CEST_C0 CEST_TLAST
1            NA  70745987 15.0       NA    6.425         NA      NA         NA
2      26908935  70745987 18.0 4.780859       NA         NA      NA         NA
3      26908936  70745987 21.0 3.498742       NA         NA      NA         NA
4      26908937  70745987 24.0 2.560460       NA         NA      NA   2.560460
7            NA  70745987 26.0       NA       NA 2.07932333      NA         NA
6            NA  70745987 30.0       NA       NA 1.37129241      NA         NA
5            NA  70745987 48.0       NA       NA 0.21065234      NA         NA
11           NA 130509330 15.0       NA    6.935         NA      NA         NA
21     26908954 130509330 18.0 4.124984       NA         NA      NA         NA
31     26908955 130509330 21.0 3.017955       NA         NA      NA         NA
41     26908956 130509330 24.0 2.208021       NA         NA      NA   2.208021
71           NA 130509330 26.0       NA       NA 1.79279271      NA         NA
61           NA 130509330 30.0       NA       NA 1.18190885      NA         NA
51           NA 130509330 48.0       NA       NA 0.18127026      NA         NA
12           NA 246409536 15.0       NA    6.115         NA      NA         NA
22     26908535 246409536 16.0 5.333360       NA         NA      NA         NA
32     26908536 246409536 18.0 4.376376       NA         NA      NA         NA
42     26908537 246409536 21.0 3.253009       NA         NA      NA         NA
52     26908538 246409536 24.0 2.417997       NA         NA      NA   2.417997
8            NA 246409536 26.0       NA       NA 1.98412710      NA         NA
72           NA 246409536 30.0       NA       NA 1.33597065      NA         NA
62           NA 246409536 48.0       NA       NA 0.22533069      NA         NA
13           NA 249265153 15.0       NA    6.185         NA      NA         NA
23     26908877 249265153 16.0 4.922666       NA         NA      NA         NA
33     26908878 249265153 18.0 3.780412       NA         NA      NA         NA
43     26908879 249265153 21.0 2.544177       NA         NA      NA         NA
53     26908880 249265153 24.0 1.712204       NA         NA      NA   1.712204
81           NA 249265153 26.0       NA       NA 1.31490467      NA         NA
73           NA 249265153 30.0       NA       NA 0.77548221      NA         NA
63           NA 249265153 48.0       NA       NA 0.07204783      NA         NA
```
